gadgets/top_blockio/test/top_file_blockio.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2024 The Inspektor Gadget authors
```

Comment:
`gadgets/top_blockio/test/top_file_blockio.go` -> `gadgets/top_blockio/test/top_blockio_test.go`. In fact, test is not even being run.